docs/reference/commandline/build.md

```diff
@@ -300,3 +301,11 @@ like `ENV` values do.
 
 For detailed information on using `ARG` and `ENV` instructions, see the
 [Dockerfile reference](../builder.md).
+
+### Specify isolation technology for container (--isolation)
```

Comment:
Can you file an issue against me to change the code and leave this as a docs PR.